src/App.vue

```diff
@@ -18,7 +18,9 @@
   }
 }
 
-
+.theme--dark.v-text-field--solo-inverted.v-input--is-focused > .v-input__control > .v-input__slot input {
```

Comment:
there has got to be a better way to style that

Reply:
Maybe but I couldnt find one

Reply:
True, good point